### PR TITLE
Add account settings to the identity header

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?
Updates the identity header to be more identity-focused with a `my account` dropdown, and also use css & js under the identity project. 

At the moment the identity header is only visible in `/consents` and `/reauthenticate` but the short term plan is to enable it for all account settings pages after everything is fine and working

**Review thingies!** There's a bit of code duplication (because, after all, it is another header) which i tried to avoid as much as possible (see `DropdownMenus.scala` but perhaps we could have a global js dropdown helper class?

## What is the value of this and can you measure success?
The new gu.com header doesn't really work for profile pages (nor should, identity is sort of its own thing) as it brings the global navigation in while hiding the account links (change pwd, user profile) under the veggie burger on mobile and outright takes them away on desktop.

This also helps towards our long term goal of splitting the identity pages from `frontend` into `identity-frontend` by removing global dependencies.

## Screenshots
screen shot 2017-12-19 at 17 33 57
screen shot 2017-12-19 at 17 34 02

## Tested in CODE?
Not yet

### CODE REVIEW

File: common/app/views/support/DropdownMenus.scala [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      label = "Change password"
+    ),
+    DropdownMenuItem(
+      href =Some(s"${Configuration.id.url}/signout"),
+      dataName = Some("nav2 : topbar : sign out"),
+      label = "Sign out"
+    )
+  )
+
+}
```
Comment (TBonnin) [inline common/app/views/support/DropdownMenus.scala]: Does this file need to live in `/common`?

Comment (walaura) [reply common/app/views/support/DropdownMenus.scala]: @TBonnin this menu is also used in the [global header](https://github.com/guardian/frontend/pull/18550/files#diff-ddd37e183676780af1a76b20b62b5d2c) (and the mobile sidebar, but i didn't apply it there) too. It's duplicated in a couple places and it felt like a useful reusable bit, plus the file structure can be used for the rest of dropdowns

Comment (TBonnin) [reply common/app/views/support/DropdownMenus.scala]: sounds good

Comment (walaura) [reply common/app/views/support/DropdownMenus.scala]: @TBonnin looking at this with fresh morning eyes i could apply it on the mobile sidebar on the scope of this PR, right now it's a it unclear what the benefits to this file are. Is that okay? didn't want to step on frontend turf too much 😅

**Comment by TBonnin** [comment]
It is up to you. If it reused (or could be reused by multiple apps) `common` is the right place for it. However if it is only usefull in the scope of `identity` it is probably better to keep it there.

**Review by Calum-Campbell — APPROVED** [review]
With my limited frontend knowledge generally LGTM 👍

**Comment by zeftilldeath** [comment]
Froma visual standpoint it will need some changes early next year and maybe we can even share the same code for drop-down menus like that. But for the time being looks great!

**Pull request merged**